sdk/security_keyvault/examples/get_secret.rs

```diff
@@ -18,9 +18,9 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         client_secret,
         TokenCredentialOptions::default(),
     ));
-    let client = KeyvaultClient::new(&keyvault_url, creds)?.secret_client(secret_name);
+    let client = SecretClient::new(&keyvault_url, creds)?;
```

Comment:
So what does setting options look like now with a builder pattern? When I last updated this crate, it was using the `ClientOptions` I originally proposed.

Reply:
Per the discussion at SDK meeting, the builder pattern for creating clients is a work in progress by @yoshuawuyts .  We'll update this crate once we have flushed out that design.